src/app/sort/sort.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit, Output } from "@angular/core";
+import { EventEmitter } from "@angular/core";
+
+interface Options {
```

Comment:
I recommend you to extract this to another file and give more context on the naming, because like this I don't know it's part of the sorting context.